Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.20.10 AS builder
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+
+WORKDIR /qe-tools
+
+RUN git clone --branch main https://github.com/redhat-appstudio/qe-tools.git . && \
+    CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o qe-tools main.go
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3
+COPY --from=builder /qe-tools/qe-tools /usr/bin/
```

Comment:
we need to copy the `config` folder as well, so the commands like `estimate` and `health-check` can load the default configuration